charts/sonarqube-dce/templates/sonarqube-search.yaml

```diff
@@ -183,9 +183,9 @@ spec:
           volumeMounts:
             - name: init-fs
               mountPath: /tmp/scripts/
-{{- if .Values.searchNodes.persistence.mounts }}
-{{ toYaml .Values.searchNodes.persistence.mounts | indent 12 }}
-{{- end }}
+            {{- if .Values.searchNodes.persistence.mounts }}
+            {{- toYaml .Values.searchNodes.persistence.mounts | indent 12 }}
```

Comment:
Do we need the `-` here?

Reply:
Good catch, yes it is needed but i forgot to change `indent` into `nindent` as per this [issue](https://github.com/helm/helm/issues/3854)

The goal was to have the `if/else` also indented for clarity, which requires to chomp space on the left